app/script/auth/module/action/RuntimeAction.js

```diff
@@ -17,12 +17,18 @@
  *
  */
 
+import * as Environment from '../../Environment';
 import * as RuntimeActionCreator from './creator/RuntimeActionCreator';
-import {isFirefox, isSupportedBrowser} from '../../Runtime';
+import {isFirefox, isSupportedBrowser, isMobileOs, isSafari} from '../../Runtime';
 
 export function checkSupportedBrowser() {
   return function(dispatch) {
-    if (isSupportedBrowser()) {
+    const isPwaSupportedBrowser = Environment.onEnvironment(
```

Comment:
Let's make `Environment.onEnvironment()` more verbose in the future by using an object as param that specifies `local`, `staging`, `prod`.